src/header.py

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
This whitespace is significant.

Reply:
That's what I suspected, but it's not clear why. Is additional code added in front of it?

Reply:
Yes, these files are compiled into the single-file `versioneer.py` file.

Reply:
The alternative would be to add the extra line when compiling the `versioneer.py` file in `setup.py`. But then, why change something that works?

Reply:
Fixed, but I still need to look into the files under `test`.